openslides/motion/forms.py

```diff
@@ -11,7 +11,7 @@
 
 from ckeditor.widgets import CKEditorWidget
 
-from .models import Category, Motion, Workflow
+from .models import Category, Motion, Workflow, Tag
```

Comment:
I do this in another pull request where we integrate isort and sort all imports. See #1383